### ISSUE TITLE
OwnCloud forgets proxy settings

### ISSUE THREAD

**Issue by solopa** [owncloud/client#357]
Hi,

I think this bug is related to this one https://github.com/owncloud/mirall/issues/50 (closed because should be fixed in 1.2.0).

The problem is OwnCloud client doesn't remember the proxy settings between sessions:

Some info:
OwnCloud Windows client version 1.2.0
Windows XP SP 3 32 bits.
Two proxies used: one with authentication (system proxy) and another one without. If I use the proxy system, OwnCloud complains saying the proxy requires authentication. If I configure the proxy to use the other one (manual configuration), it works as expected. However, the next time I launch OwnCloud, the proxy settings returns to "use system proxy", and forgets the user/password I set up in the manual config. Once I put them again, it works (till the next session).

So, in short, it doesn't remember the proxy option and its configuration.

Hope you can look at this, because it's a little bit annoying.

Thanks!
